projects/m1/017-what-day-of-the-week/solutions/bisicchiamatteo/017-what-day-of-the-week.py

```diff
@@ -0,0 +1,12 @@
+#inserimento anno  
+CONF_YEAR=1000
+POS_DAYS=5
+days_week= ["Sunday","Monday","Tuesday","Wednesday" ,"Thursday","Friday","Saturday"]
+year=int(input("inserisci l'anno :"))
+val_giorno=int(((year-CONF_YEAR)+((year-CONF_YEAR)/4))%7)
```

Comment:
Per favore @ergirius , abituati a usare nomi di variabili in inglese il più possibile